docs/concepts/cluster-administration/cloud-providers.md

```diff
@@ -58,50 +58,108 @@ The information for the annotations for AWS is taken from the comments on [aws.g
 
 # OpenStack
 This section describes all the possible configurations which can
-be used when using OpenStack with Kubernetes.
+be used when using OpenStack with Kubernetes. The OpenStack cloud provider
+implementation for Kubernetes supports the use of these OpenStack services from
+the underlying cloud, where available:
+
+Service                  | API Version(s) | Required
+-------------------------+----------------+----------
+Block Storage (Cinder)   | V1†, V2        | No
+Compute (Nova)           | V2             | No
+Identity (Keystone)      | V2, V3         | Yes
```

Comment:
Keystone V2 will be removed in the next OpenStack release #52563

Reply:
Ack will note this. I imagine like some of the others on our end we will need to keep this support for a while as we don't yet expose any disclaimers about what OpenStack versions a given K8S release will work with. We probably need to start thinking about it though as we move to drop things.

Reply:
Marking endpoints as deprecated and scheduled for removal helps send a clear message about maintaining installations and interoperabaility.

Reply:
Yes but doesn't address the other end of the problem I am referring to. E.g. for a given K8S release (1.8) what versions of OpenStack are supported? Based on the listed API versions at least Liberty but not clear whether or not that is  actually true in practice.

If Keystone will drop V2 in Queens, when will we drop K8S support for OSP releases < Queens? That is the discussion we need to have.